src/vs/workbench/parts/output/electron-browser/outputServices.ts

```diff
@@ -612,9 +601,7 @@ export class OutputService extends Disposable implements IOutputService, ITextMo
 					if (!preserveFocus) {
 						this._outputPanel.focus();
 					}
-				})
-				// Activate smart scroll when switching back to the output panel
-				.then(() => this.setPrimaryCursorToLastLine());
```

Comment:
The reason we probably called this function was to resume scrolling. Now we shouldn't call it anymore because scrolling depends only on the lock action state, and the lock state should persist between channel changes.